teamcity.build

```diff
@@ -4,6 +4,7 @@
 	<property name="root.dir" value="." />
 	<property name="config.teamcity" value="default"/>
 	<property name="current-test-configuration.dir" value="${root.dir}/current-test-configuration" /> 
+	<property name="skip.db-service" value="true" overwrite="false" />
```

Comment:
Another way of enabling the "Start/Stop" change would be to directly change the default value here to `false`.
But that looks less convenient to me. Having to enable this in the build template instead would allow to wait for having this change merged in most branches first. Then enabling it would not cause any of those branches to fail building.

While on the contrary, enabling it by source code change will cause it to make all other branches failing until they get the source code change.

Changing its default value here would likely call for more changes in `ShowBuildMenu.bat` too: personally I do not have the same services name than the TeamCity build agent. So for the TeamCity builds to run through the menu, `ShowBuildMenu.bat` would have to provide this property as `true`.